### DISCUSSION TITLE
Everything okay with @xeffyr?

### DISCUSSION THREAD

**Discussion by finagolfin** [termux/termux-packages#8904] [Q&A] [answered]
His github account has been deleted and his gmail account says it doesn't exist.

**Comment by thunder-coding** [comment, marked as answer]
There already has been a discussion regarding this on Gitter. Most probably it's war tentions in their country which made them take this decision

https://gitter.im/termux/dev?at=61f9f43151bd8b580c82459b

All we can do is hope that the situations become better soon.

**Reply by cujoXavier** [reply]
My Thoughts, Hopes && prayers are with them && their country. Tyvm 4 answering butterfaces' question despite the discussion on gitter having addressed it already. 
Above & beyond as per usual there τΉυηδεя-ςøð¡ηφ

**Reply by T-Dynamos** [reply]
Is he back ?
https://github.com/sylirre

**Reply by 2096779623** [reply]
> Is he back ? https://github.com/sylirre

Yes.

>xeffyr don't have to wait for the war to end in order to join back the organisation. If they just mail anyone of the current members showing interest in joining the organisation again, surely they'll be welcomed.

**Reply by T-Dynamos** [reply]
Yeah , I am excited :) .

**Reply by 2096779623** [reply]
> Yeah , I am excited :) .

Me too :) .

**Comment by 2096779623** [comment]
xeffyr(sylirre) is back and this discussion is over!
@sylirre
